vignettes/admiral.Rmd

```diff
@@ -76,6 +81,50 @@ The output dataset is ungrouped. The observations are not ordered in a dedicated
 way. In particular, the order of the observations of the input dataset may not
 be preserved.
 
+# Computations
+
+[Computations](../reference/index.html#section-computations) expect vectors as
+input and return a vector. These function can be used in expressions like
+`convert_dtc_to_dt()` in the derivation of `FINLABDT` in the
+example below:
+
+```{r, eval=TRUE}
+# derive final lab visit date
+ds_final_lab_visit <- ds %>% 
+  filter(DSDECOD == "FINAL LAB VISIT") %>% 
+  transmute(USUBJID, FINLABDT = convert_dtc_to_dt(DSSTDTC))
+
+# derive treatment variables  
+adsl <- dm %>%
+  mutate(TRT01P = ARMCD, TRT01A = ACTARMCD) %>%
+  derive_var_trtsdtm(dataset_ex = ex) %>%
+  derive_var_trtedtm(dataset_ex = ex) %>%
+  derive_vars_dtm_to_dt(vars(TRTSDTM, TRTEDTM)) %>%
+  derive_var_trtdurd() %>%
+  # merge on final lab visit date
+  left_join(ds_final_lab_visit, by = "USUBJID")
+```
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(adsl)
+```
+# Parameters
```

Comment:
This is not rendered correctly. I think there needs to be an empty line before a new section.

Reply:
Fixed.

Reply:
Done.

Reply:
Done.